### PR TITLE
Added context ID to all logs

### PR DESCRIPTION
**Pull request by amobarak**
All log scenarios:
image

Each scenario is commented.
Resolves: https://companieshouse.atlassian.net/browse/DSND-636

### CODE REVIEW

File: src/main/java/uk/gov/companieshouse/insolvency/delta/consumer/InsolvencyDeltaConsumer.java [hunk 1]
```diff
@@ -54,8 +54,10 @@ public void receiveMainMessages(Message<ChsDelta> message,
                                     @Header(KafkaHeaders.RECEIVED_TOPIC) String topic,
                                     @Header(KafkaHeaders.RECEIVED_PARTITION_ID) String partition,
                                     @Header(KafkaHeaders.OFFSET) String offset) {
-        logger.trace(String.format("A new message from %s topic with payload:%s "
-                + "and headers:%s ", topic, message.getPayload(), message.getHeaders()));
+        logger.trace(String.format("A new message successfully picked up "
```
Comment (amobarak) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/consumer/InsolvencyDeltaConsumer.java:57]: image

File: src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java [hunk 2]
```diff
@@ -71,18 +75,15 @@ public void processDelta(Message<ChsDelta> chsDelta,
 
         InternalCompanyInsolvency internalCompanyInsolvency = transformer.transform(insolvency);
 
+        logger.trace(String.format("Message with contextId: %s successfully "
```
Comment (amobarak) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java:78]: image

File: src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java [hunk 3]
```diff
@@ -149,11 +145,17 @@ private void handleResponse(
                     .format("Bad request PUT Api Response %s", msg));
         } else if (!httpStatus.is2xxSuccessful()) {
             // any other client or server status is retryable
-            logger.errorContext(logContext, msg + ", retry", null, logMap);
+            logger.trace(String.format("Failed to invoke insolvency-data-api "
```
Comment (amobarak) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java:148]: image

File: src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java [hunk 4]
```diff
@@ -173,11 +175,17 @@ private void handleDeleteResponse(
...
+                            + "DELETE endpoint for message with contextId: %s "
+                            + "with error %s",
+                    logContext,
+                    msg));
             throw new RetryableErrorException(
                     String.format("Unsuccessful DELETE API response, %s", msg));
         } else {
-            logger.trace("Got success response from DELETE insolvency request");
+            logger.trace(String.format("Successfully invoked insolvency-data-api "
```
Comment (amobarak) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java:186]: image

File: src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java [hunk 5]
```diff
@@ -173,11 +175,17 @@ private void handleDeleteResponse(
                     String.format("Bad request DELETE Api Response %s", msg));
         } else if (!httpStatus.is2xxSuccessful()) {
             // any other client or server status is retryable
-            logger.errorContext(logContext, msg + ", retry", null, logMap);
+            logger.trace(String.format("Failed to invoke insolvency-data-api "
```
Comment (amobarak) [inline src/main/java/uk/gov/companieshouse/insolvency/delta/processor/InsolvencyDeltaProcessor.java:178]: image

**Review by zahidm3 — APPROVED** [review]

**Review by iichr — APPROVED** [review]

**Review by gseregni-ch — APPROVED** [review]

**Pull request merged**